src/main.cpp

```diff
@@ -508,9 +507,7 @@ bool FullSyncWithDPORNodes()
             return true;
 }
 
-
-
-double GetPoSKernelPS2()
+double GetPoSKernelPS()
```

Comment:
I think we should move this to kernel.cpp/h, right?

Reply:
I don't think so. Kernel.cpp is supposed to contain Functions to calculate and check kernel components. This is just stats gathering function.

Reply:
Gotcha!